pkg/kudoctl/cmd/init.go

```diff
@@ -155,3 +174,40 @@ func (i *initCmd) YAMLWriter(w io.Writer, manifests []string) error {
 	_, err := fmt.Fprintln(w, "...")
 	return err
 }
+
+//func initialize(fs afero.Fs, settings env.Settings, out io.Writer) error {
+func (i *initCmd) initialize() error {
+
+	if err := ensureDirectories(i.fs, i.home, i.out); err != nil {
+		return err
+	}
+
+	return ensureRepositoryFile(i.fs, i.home, i.out)
+}
+
+func ensureRepositoryFile(fs afero.Fs, home kudohome.Home, out io.Writer) error {
+	if !files.Exists(fs, home.RepositoryFile()) {
+		fmt.Fprintf(out, "Creating %s \n", home.RepositoryFile())
+		r := repo.NewRepoFile()
+		if err := r.WriteFile(fs, home.RepositoryFile(), 0644); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func ensureDirectories(fs afero.Fs, home kudohome.Home, out io.Writer) error {
+	dirs := []string{
+		home.String(),
```

Comment:
home.Path?

Reply:
String is ok here, it adheres it to the Stringer interface which could be useful down the road.